KaitaiStream.cs

```diff
@@ -7,67 +7,89 @@
 namespace Kaitai
 {
     /// <summary>
-    /// The base Kaitai steam which exposes an API for the Kaitai Struct framework.
+    /// The base Kaitai stream which exposes an API for the Kaitai Struct framework.
     /// It's based off a <code>BinaryReader</code>, which is a little-endian reader.
     /// </summary>
     public partial class KaitaiStream : BinaryReader
     {
         #region Constructors
 
+        /// <summary>
+        /// Create a KaitaiStream backed by abstract stream. It could be in-memory buffer or open file.
+        /// </summary>
         public KaitaiStream(Stream stream) : base(stream)
         {
         }
 
-        ///<summary>
-        /// Creates a KaitaiStream backed by a file (RO)
-        ///</summary>
-        public KaitaiStream(string file) : base(File.Open(file, FileMode.Open, FileAccess.Read, FileShare.Read))
+        /// <summary>
+        /// Create a KaitaiStream backed by a closed file (in read-only binary-mode).
```

Comment:
Why it's "closed file"? It actually remains open.

Reply:
Its a less known Python terminology. In general, functions that work on files are divided into those that operate on (currently) closed files and take filenames and those that operate on (currently) open files and take integer file descriptors/handles.

Reply:
Fixed.